Add workers extraVolumes to Kubernetes pod template for Helm Chart

loustler opened the pull request:
As far as I know, the pod_template is a worker template for creating workers by Kubernetes Executor.
We already have the extraVolumes value for workers, I think it can apply to the pod_template.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

loustler committed:
Add workers extraVolumes to Kubernetes pod template for Helm Chart

Pod template is worker created by Kubernetes Executor.
We already have the extraVolumes value for workers, it can apply to the pod template.

ianstanton reviewed (commented):
@loustler Looks good. Please add tests as well. cc @jedcunningham

loustler committed:
Add extraVolumes and extraVolumeMounts test for pod-template-file

Test given worker's extraVolumes and extraVolumeMounts should create valid pod-template-file

loustler committed:
Fix test again

Fix test again

loustler committed:
FIx test

Fix test again

loustler committed:
Fix lint error

Fix lint error in test code

loustler committed:
Fix lint

Fix lint
